source/mqtt/MqttClient.cpp

```diff
@@ -698,6 +700,21 @@ namespace Aws
                 return packetId;
             }
 
+            const MqttConnectionOperationStatistics &MqttConnection::GetOperationStatistics() noexcept
+            {
+                if (m_underlyingConnection != nullptr)
+                {
+                    aws_mqtt_client_connection_get_stats(m_underlyingConnection, &m_operationStatisticsNative);
```

Comment:
A minor suggestions is that I feel it is not necessary to make the operationStatisticsNative as a class member as we only use the data once here. 
We could do something like:
```
aws_mqtt5_client_operation_statistics operationStatisticsNative = {0,0,0,0};
aws_mqtt_client_connection_get_stats(m_underlyingConnection, &operationStatisticsNative);
```

Reply:
Good idea! I'll adjust it